examples/solidity/basic/default.yaml

```diff
@@ -68,4 +68,5 @@ filterFunctions: []
 filterBlacklist: true
 #directory to save the corpus; by default is disabled  
 corpusDir: null
-
+# constants for corpus mutations (for experimentation only)
+mutConsts: [1, 1, 1]
```

Comment:
let's make these constants more descriptive, e.g.
```yaml
weightMutateXXX: 1
weightMutateYYY: 1
weightMutateZZZ: 2
```

Reply:
This will grow in the near feature (check the other PRs with more mutations). I don't think that will be a good idea.